gpu-service/jobs.py

```
@@ -0,0 +1,288 @@
+            metadata = {}
+        
+        # Process image with face recognition
+        with tempfile.NamedTemporaryFile(suffix=os.path.splitext(object_name)[1], delete=True) as tmp_image:
+            tmp_image.write(image_bytes)
+            tmp_image.flush()
+
+            image = face_recognition.load_image_file(tmp_image.name)
+            faces = RetinaFace.detect_faces(tmp_image.name)
+
+            if not isinstance(faces, dict):
+                logger.info(f"No faces detected in {object_name}")
```

Ruff (0.11.9)

Logging statement uses f-string

(G004)